src/software/sensor_fusion/filter/robot_team_filter.h

```diff
@@ -23,8 +23,10 @@ class RobotTeamFilter
      *
      * @return The updated state of the team given the new data
      */
-    Team getFilteredData(const Team& current_team_state,
-                         const std::vector<RobotDetection>& new_robot_detections);
+    Team getFilteredData(
```

Comment:
Remember to update docs

Reply:
Just the header file, you just need to add a `@param` doc to the existing comments